### ISSUE TITLE
flight service is not immediately deleted when m4dapplication is created in batches

### ISSUE THREAD

**Issue by lappanna** [fybrik/fybrik#710]
Even though there were no m4dapplications, flight service deployments in `m4d-blueprints` were not deleted immediately.

```
# oc get pods -n m4d-blueprints | wc;
    101     505    9999
[root@table-inf ~]# oc get m4dapplications --all-namespaces 
No resources found
```
pods were in running state
```
# oc get pods -n m4d-blueprints
NAME                                                              READY   STATUS    RESTARTS   AGE
my-m4d-b1-287xp-m4d-notebook-sample-arrow-fligh-c78a2-arrocdr4v   1/1     Running   0          37h
my-m4d-b1-2pb8h-m4d-notebook-sample-arrow-fligh-c78a2-arrobsxkh   1/1     Running   0          37h
my-m4d-b1-2pfrs-m4d-notebook-sample-arrow-fligh-c78a2-arrott7t9   1/1     Running   0          37h
my-m4d-b1-2q2jn-m4d-notebook-sample-arrow-fligh-c78a2-arrowjwlp   1/1     Running   0          37h
my-m4d-b1-2rqfd-m4d-notebook-sample-arrow-fligh-c78a2-arro8lfrh   1/1     Running   0          37h
my-m4d-b1-2wxgc-m4d-notebook-sample-arrow-fligh-c78a2-arrok8p6h   1/1     Running   0          37h
my-m4d-b1-4nc78-m4d-notebook-sample-arrow-fligh-c78a2-arrozshkw   1/1     Running   0          36h
my-m4d-b1-4qnmb-m4d-notebook-sample-arrow-fligh-c78a2-arroq4fjk   1/1     Running   0          37h
my-m4d-b1-54cn8-m4d-notebook-sample-arrow-fligh-c78a2-arro5hsvn   1/1     Running   0          37h
```

follow the steps given in https://github.com/mesh-for-data/mesh-for-data/issues/708 to create large number of m4dapplications and then delete them using 
```
oc delete m4dapplications -n m4d-notebook-sample --all
```

**Comment by shlomitk1** [comment]
I've created 1000 fybrikapplications (read scenario). After blueprints were created together with the relevant pods, I've deleted all fybrikapplications.
Here are my observations:
- FybrikApplications and Plotters are deleted after some (short) time.
- Blueprints are deleted slow (~14 blueprints per minute)
- No significant difference between pod deletion and blueprint deletion.

Need to check how to increase blueprint deletion performance.
